src/dosbox.cpp

```diff
@@ -837,6 +837,11 @@ void DOSBOX_Init(void) {
 	Pbool = secprop->Add_bool("umb",Property::Changeable::WhenIdle,true);
 	Pbool->Set_help("Enable UMB support.");
 
+	pstring = secprop->Add_string("ver", when_idle, "");
+	pstring->Set_help("Set DOS version (5.0 by default). Specify as major.minor format.\n"
```

Comment:
If it's 5.0 by default, then please write `5.0` as default value instead of an empty string.

Reply:
Done.